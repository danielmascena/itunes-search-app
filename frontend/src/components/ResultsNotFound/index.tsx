import * as React from 'react';

const ResultsNotFound: React.FC = () => (
    <div>You got zero results. Try again changing the term of your search.</div>
);

export default ResultsNotFound;