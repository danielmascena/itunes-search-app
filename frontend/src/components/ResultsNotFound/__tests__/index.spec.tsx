import * as React from "react";
import renderer from 'react-test-renderer';

import ResultsNotFound from "../index";

test("should renders the ResultsNotFound component", () => {
    expect(renderer.create(<ResultsNotFound />).toJSON()).toMatchSnapshot();
});