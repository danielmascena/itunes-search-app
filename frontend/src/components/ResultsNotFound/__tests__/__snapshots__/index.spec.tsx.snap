// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders the ResultsNotFound component 1`] = `
<div>
  You got zero results. Try again changing the term of your search.
</div>
`;
